src/wwwroot/template-editor/adapterService.js

```diff
@@ -112,9 +112,9 @@
         lastModifiedDate: '2015-11-14T06:35:42.239Z',
         size: 1024,
         type: '.jpg',
-        url: 'http://cdn.fromdoppler.com/mseditor/v1.0.0-build306/uploads/fakeImage1.jpg',
-        thumbnailUrl: 'http://cdn.fromdoppler.com/mseditor/v1.0.0-build306/uploads/fakeImage1.jpg',
-        thumbnailUrl150: 'http://cdn.fromdoppler.com/mseditor/v1.0.0-build306/uploads/fakeImage1.jpg'
+        url: 'http://cdn.fromdoppler.com/mseditor/relay-demo-content/fakeImage1.jpg',
+        thumbnailUrl: 'http://cdn.fromdoppler.com/mseditor/relay-demo-content/fakeImage1.jpg',
+        thumbnailUrl150: 'http://cdn.fromdoppler.com/mseditor/relay-demo-content/fakeImage1.jpg'
```

Comment:
```
http://cdn.fromdoppler.com/mseditor/relay-demo-content
                           ˆˆˆˆˆˆˆˆ
```

In my opinion, it has nothing to do with MSEditor, it is simply something added for Relay. Am I missing something?